utils/python/CIME/XML/generic_xml.py

```diff
@@ -125,21 +137,12 @@ def add_child(self, node, root=None):
             root = self.root
         self.root.append(node)
 
-    def get_value(self, item, resolved=True):
+    def get_value(self, item, resolved=True, settype=True):
         """
         get_value is expected to be defined by the derived classes, if you get here it is an error.
         """
         logger.debug("Get Value for "+item)
-        result = None
```

Comment:
Changed comment to "get_value is expected to be defined by the derived classes, if you get here 
        the value was not found in the class."